htdocs/projet/tasks/time.php

```diff
@@ -1047,12 +1043,12 @@
 
 			// Description
 			print '<td class="titlefield tdtop">'.$langs->trans("Description").'</td><td>';
-			print dol_htmlentitiesbr($projectstatic->description);
+			print nl2br($projectstatic->description);
```

Comment:
Why replacing with nl2br ?
description is text of html, and to ouput this on screen we should use
dol_htmlentitiesbr

Reply:
True, fixed